src/vcpkg/commands.download.cpp

```diff
@@ -82,7 +82,6 @@ namespace vcpkg
             {
                 Checks::msg_exit_with_error(VCPKG_LINE_INFO, msgImproperShaLength, msg::value = *p);
             }
-            Strings::inplace_ascii_to_lowercase(p->data(), p->data() + p->size());
```

Comment:
I think it's safe to make this change unconditionally because we already did it 99% of the time.